zarr/indexing.py

```diff
@@ -691,11 +691,7 @@ def __init__(self, selection, array):
         self.chunk_rixs = np.nonzero(self.chunk_nitems)[0]
 
         # unravel chunk indices
-        if tuple(map(int, np.__version__.split('.')[:2])) < (1, 16):
-            self.chunk_mixs = np.unravel_index(self.chunk_rixs, dims=array._cdata_shape)
-        else:
-            # deal with change dims->shape in arguments as of numpy 1.16
-            self.chunk_mixs = np.unravel_index(self.chunk_rixs, shape=array._cdata_shape)
+        self.chunk_mixs = np.unravel_index(self.chunk_rixs, array._cdata_shape)
```

Comment:
Do you know if the keyword argument was needed for some reason @alimanfoo? Noticed this change came from [here]( https://github.com/zarr-developers/zarr-python/pull/468/files#diff-927f13103811bc87516c2e8e88776ce5 ). Hence the question 🙂

Reply:
In general, I think using kwargs is better when it's defined as a kwarg rather than relying on the order of positional args. The previous block looked like it was here to deal with numpy <1.16 and numpy >1.16 but in our `setup.py` we specify `numpy>=1.17`, so I think it's safe to drop handling old `numpy` versions and use `shape=` but not fussed too much either way.

Reply:
IIUC our `setup.py` has NumPy 1.7 as the minimum supported version

https://github.com/zarr-developers/zarr-python/blob/3ae943914c5241b7e2e4c2ee730a23344d5299be/setup.py#L12-L17

Reply:
Yep. :)

Reply:
So I guess we do need to consider older versions of NumPy where the signature keywords may have been different

Reply:
Ah, I kept reading 1.7 as 1.17... 🤦‍♂

Reply:
Seems to be largely unchanged over time. 🙂 Here are [the docs from NumPy 1.7]( https://docs.scipy.org/doc/numpy-1.7.0/reference/generated/numpy.unravel_index.html ).

Reply:
Cool, cool. Happy for this to be merged on my side :)